examples/llm/yamls/mosaic_gpt/13b.yaml

```diff
@@ -5,12 +5,12 @@ max_seq_len: 2048
 global_seed: 17
 
 # Run Name
-run_name: gpt-13b
+run_name: null
```

Comment:
Ok, I'm going to stop adding that comment because there's like 20 files that should have that text next to `run_name`. But, you get the idea.